src/f_op/f_op_msg_mng.cpp

```diff
@@ -922,7 +935,7 @@ void fopMsgM_outFontSet(J2DPicture* param_1, J2DPicture* param_2, s16* param_3,
     param_2->show();
     fopMsgM_blendDraw(param_1, fopMsgM_buttonTex[param_5]);
     fopMsgM_blendDraw(param_2, fopMsgM_buttonTex[param_5]);
-    if((0xA <= param_5 && param_5 <= 0xE) || param_5 == 0x15 || param_5 == 0x17) {
+    if((u8)(param_5 - 10) <= 3 || param_5 == 0x15 || param_5 == 0x17) {
```

Comment:
Usually when you see `(unsigned cast)(value - base) <= range` it's a compiler optimization for a chain of OR statements checking contiguous values against a single variable (especially since this parameter had a high chance of being an enum originally). Does this still match?
```suggestion
    if(param_5 == 0xA || param_5 == 0xB || param_5 == 0xC || param_5 == 0xD || param_5 == 0x15 || param_5 == 0x17) {
```

Reply:
Aah yeah it still matches with that, thanks. Good to know.